pkg/parse/parse_unix.go

```diff
@@ -19,3 +23,19 @@ func getDefaultProcessLimits() []string {
 	}
 	return defaultLimits
 }
+
+func DeviceFromPath(device string) (configs.Device, error) {
+	src, dst, permissions, err := Device(device)
+	if err != nil {
+		return configs.Device{}, err
+	}
+	if unshare.IsRootless() {
+		return configs.Device{}, errors.Errorf("Renameing device %s to %s is not a supported in rootless containers", src, dst)
```

Comment:
`s/Renameing/Renaming/` ? It predates this PR but as we're already on it :)

Reply:
Fixed.